paypal/paypal/helper.php

```diff
@@ -88,7 +88,9 @@ public function Storelog($name, $logdata)
 	 */
 	public function validateIPN($data, $componentName)
 	{
-		$url              = self::buildPaypalUrl();
+		$sandBoxMode = $this->params->get('sandbox', '0', 'INT');
+		$url = ($sandBoxMode)?'https://ipnpb.sandbox.paypal.com/cgi-bin/webscr':'https://ipnpb.paypal.com/cgi-bin/webscr';
```

Comment:
Can you make it more readable like
`$url = ($sandBoxMode) ? 'https://ipnpb.sandbox.paypal.com/cgi-bin/webscr' : 'https://ipnpb.paypal.com/cgi-bin/webscr';`

@ankush-maherwal

Reply:
done